fe1-web/docs/README.md

```diff
@@ -47,13 +47,20 @@ Here's the annotated directory tree:
 │
 │   ├── components               # library of simple, reusable UI component
 │   │
+│   ├── contexts                 # feature context type definition
+│   │
+│   ├── functions                # module containing functions that are useful in the whole application
+│   │
+│   ├── hooks                    # contains a wrapper for a useful react hook
```

Comment:
Not sure whether we should be this specific in the documentation as maybe there will be multiple ones in this directory later?

Reply:
not sure I understand your comment @Tyratox . Do you mean the following ?
```suggestion
│   ├── hooks                    # contains utility react hooks
```

Reply:
True!